### ISSUE TITLE
Baseline (sourceonly ResNet) miou is 36.6

### ISSUE THREAD

**Issue by SpinachR** [wasidennis/AdaptSegNet#35]
The sourceonly resnet-101 performance is quite higher than mine, which is only ~27.

I have noticed that the parameters in batch_norm layer are not trained. 
Do we need to update the batch statistics during training or just use the statistics load from the pre-trained resnet?

Could you provide more details based on resnet-101 model? @wasidennis

**Comment by wasidennis** [comment]
Since we are using the batch size as 1, we turned off the batch norm updates.

**Comment by alphjheon** [comment]
@wasidennis  In case that we would like to use Batch normalization, how can we switch it on?

**Comment by wasidennis** [comment]
@alphjheon In the model, you can comment out the lines regarding setting BN layers "not required gradients", like the following:

for i in self.bn3.parameters():
            i.requires_grad = False

**Comment by tarun005** [comment]
@SpinachR My baseline ResNet-101 trained on GTA only is also around ~29%. The paper however reports ~35%. Can you tell steps to reproduce source only performance?
